src/data/textparsing.py

```diff
@@ -70,4 +71,13 @@ def clean_doc(doc, spacynlp):
             else:
                 doc = doc.replace(match, mergedWord)
     
+    
+    #De-dup for PUBMED articles, where the main text is sometimes duplicated
+    sliceText = doc[0:500]

```

Comment:
Why the hard-coded indexing? Is this necessary? I'm not totally clear on the `type` of `doc`, but will assume that this works.